src/Components/Sidebar/User.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import Image from '../../pages/Login/Image/Image';
+const User = () => {
+    const ProfileImageProps = {
+        src: "/images/avatars/karl.jpg",
+        alt: "img profile",
+        imageWrapperStyle: "w-16 flex mr-3",
+        imageStyle: "rounded-full"
+      };
+  return (
+    <Link to={`/p/karl`} className="grid grid-cols-4 gap-4 mb-6 items-center">
```

Comment:
This path also exists in Header.js as a hard-coded value.
If you will not be adding a login/signup functionality (backend) then it's okay you can leave it. 
But if want to link with the backend, this should be handled. 
If so, do you have a plan to handle it?